### ISSUE TITLE
HELP!Compile tensorflow source code WITH cuda10.0(2080ti),CUDNN 7.4.1,bazel0.15 anaconda2

### ISSUE THREAD

**Issue by xiehousen** [tensorflow/tensorflow#24176]
ERROR: /home/xhs/XHS/tensorflow-master/tensorflow/BUILD:573:1: Executing genrule //tensorflow:tf_python_api_gen_v1 failed (Exit 1)
Traceback (most recent call last):
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/tools/api/generator/create_python_api.py", line 27, in <module>
    from tensorflow.python.tools.api.generator import doc_srcs
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/__init__.py", line 81, in <module>
    from tensorflow.python import keras
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/keras/__init__.py", line 29, in <module>
    from tensorflow.python.keras import datasets
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/keras/datasets/__init__.py", line 25, in <module>
    from tensorflow.python.keras.datasets import imdb
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/keras/datasets/imdb.py", line 25, in <module>
    from tensorflow.python.keras.preprocessing.sequence import _remove_long_seq
  File "/home/xhs/.cache/bazel/_bazel_xhs/1843eb251a57c74a19c1281e427cacbd/execroot/org_tensorflow/bazel-out/host/bin/tensorflow/create_tensorflow.python_api_1_tf_python_api_gen_v1.runfiles/org_tensorflow/tensorflow/python/keras/preprocessing/__init__.py", line 21, in <module>
    import keras_preprocessing
ImportError: No module named keras_preprocessing
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 6674.103s, Critical Path: 2867.88s
INFO: 10631 processes: 10631 local.
FAILED: Build did NOT complete successfully

This is the part I chose to get the error. Please help me to see how it can be compiled.

**Comment by pldelisle** [comment]
You have to install keras_preprocessing in your virtual environment. 

See the first section here : https://www.tensorflow.org/install/source

**Comment by ymodak** [comment]
Tensorflow prebuilt binaries are tested against CUDA 9.0. If you want to use cuda 10, you have to build TF from sources yourself.

**Comment by xiehousen** [comment]
>     * Ubuntu
Yes , I build Tensorflow from source,but I encountered an error.
The system inforflow is described below：
ubuntu16.04
tensorflow source
Tensorflow source version : 1.12
python version 2.7
conda
bazel version 0.15
gcc version 5.4.0 
CUDA version 10.0
Cudnn version 7.4.1
GPU model RTX 2080ti,11GB

- Describe the problem：
First I installed bazel, the installed version is 0.15.0.
Download tensorflow source code by git clone --recurse-submodules https://github.com/tensorflow/tensorflow
./configure .In this step , I have chosen no except cuda related options.
Finally, I executed 
bazel build --config=opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
there was an error  mentioned above .
Thank you for your help.
